sway-lib-std/src/tx.sw

```diff
@@ -302,3 +303,10 @@ pub fn tx_output_amount(index: u64) -> u64 {
         r1: u64
     }
 }
+
+/// Get the id of the current transaction.
+pub fn tx_id() -> b256 {
+    asm() {
```

Comment:
Can this `asm` block be replaced with `read(VM_ADDRESS_ZERO)`?

Reply:
yep!